### PR TITLE
add small extracontexts ref. to context page and short paragraph to editor page

### PR DESCRIPTION
**Pull request by Laarryy**
For the editor, just letting users know that they should be starting a new editor session after they apply edits.

### CODE REVIEW

**Comment by Laarryy** [comment]
Yes, i did not add the relative links. Sue me. Also, Turbo if you'd like me to, let me know 😄

**Comment by Laarryy** [comment]
Turbo, I don't actually think there was anywhere to add relative links. If there are and you want me to, let me know

**Comment by Turbotailz** [comment]
Nope, none needed!

File: Web-Editor.md [hunk 1]
```diff
@@ -1,5 +1,7 @@
 LuckPerms features a web editor, which allows you to make changes to permissions data using a friendly interface in your web browser. It is safe, secure and easy to use. You don't need to host the page yourself - and it works for all storage types!
 
+It is important to note that after you run the applyedits command, you **must** start a new editor session to continue editing for it to work in all cases. If you are seeing a certain thing deleted every time you apply edits, it's because you did not start a new session after you ran the `applyedits` command!
+
 If you'd like more information about the technical details, see the project's [readme page](https://github.com/lucko/LuckPermsWeb/blob/master/editor/README.md).
```
Comment (Tobi406) [inline Web-Editor.md]:
```suggestion
If you'd like more information about the technical details, see the project's [readme page](https://github.com/lucko/LuckPermsWeb/blob/master/README.md).
```

**Review by Tobi406 — CHANGES_REQUESTED** [review]
If you're on it already, maybe change that link, too?

**Pull request merged**